GPU/Vulkan/DrawEngineVulkan.cpp

```diff
@@ -884,8 +884,11 @@ void DrawEngineVulkan::DoFlush() {
 			if (pipeline != lastPipeline_) {
 				vkCmdBindPipeline(cmd, VK_PIPELINE_BIND_POINT_GRAPHICS, pipeline->pipeline);
 				lastPipeline_ = pipeline;
+				if (lastPipeline_ && !lastPipeline_->useBlendConstant && pipeline->useBlendConstant) {
```

Comment:
Wait, those are equal now.  Doesn't this line need to be one higher - above `lastPipeline_ = pipeline;`?

-[Unknown]

Reply:
Yes, good catch, thanks!